### PR TITLE
Extend docu on global_active_cell_index

### PR DESCRIPTION
**Pull request by peterrum**

### CODE REVIEW

File: source/grid/tria.cc [hunk 1]
```diff
@@ -12696,7 +12696,7 @@ Triangulation<dim, spacedim>::reset_global_cell_indices()
   {
     types::global_cell_index cell_index = 0;
     for (const auto &cell : active_cell_iterators())
-      cell->set_active_cell_index(cell_index++);
+      cell->set_global_active_cell_index(cell_index++);
```
Comment (peterrum) [inline source/grid/tria.cc:12699]: ... and fix a but (which is a relict from times when I hoped we could redefine active cell indices).

File: include/deal.II/grid/tria_accessor.h [hunk 2]
```diff
@@ -3450,13 +3450,24 @@ class CellAccessor : public TriaAccessor<dim, dim, spacedim>
...
-   * Return global active cell index for an active cell.
+   * Return globally unique cell index for a non-artificial active cell. The 
+   * value is identical to active_cell_index() in the context of a serial 
+   * triangulation.
+   * 
+   * In the context of parallel triangulations, locally-owned cells are  
+   * enumerated contiguously within each subdomain of the mesh. 
+   * 
+   * If a cell-data vector has been set up with 
+   * parallel::global_active_cell_index_partitioner(), the returned index can
```
Comment (kronbichler) [inline include/deal.II/grid/tria_accessor.h:3461]: Do you want to add the triangulation name here?

Comment (peterrum) [reply include/deal.II/grid/tria_accessor.h:3461]: Yes: facepalm...

Comment (peterrum) [reply include/deal.II/grid/tria_accessor.h]: Changed!

**Review by kronbichler — APPROVED** [review]

**Pull request merged**